django/contrib/admin/filters.py

```diff
@@ -411,6 +411,11 @@ def choices(self, cl):
 
 
 class RelatedOnlyFieldListFilter(RelatedFieldListFilter):
+    def __init__(self, field, request, params, model, model_admin, field_path):
+        self.field_path = field_path
+        super(RelatedOnlyFieldListFilter, self).__init__(
```

Comment:
Did you evaluate the alternative proposed in https://github.com/django/django/compare/master...quarkness:related-only-filter-field-path?